assets/style.css

```diff
@@ -0,0 +1,559 @@
+}
+
+.head {
+	margin: 50px auto;
+	
+}
+
+.author span{
+	font-size: 14px;
+}
+.author a{
+	text-decoration: none;
+	color:black;
+	font-weight: bold;
+	font-size: 14px;
+}
+
+
+.date{
+	display:inline-block;
+}
+
+.social{
+	display:inline-block;
+	width:100%;
+	position:relative;
+}
+
+.media{
+    display:inline-block;
+    float: right;
+    margin: 0 auto;
+    list-style: none;
+}
+
+.date{
+	margin-top:12px;
+}
+
+.media li {
+    display: inline;
+    margin: 0 10px;
+}
+
+.rounded i {
+    background: lightgray;
+    padding: 12px;
+    border-radius: 50%;
+    cursor: pointer;
+    border:1px solid white;
+}
+
+.rounded i:hover {
+    background: white;
+    border: 1px solid gray;
+}
+
+.noround{
+	cursor:pointer;
+}
+
+
+
+.sidebars {
+/* 	width:20%;
+	position:absolute;
+	right:25px; */
+	float: left;
+    width:20%;
+	margin: 30px 0;
+
+}
+
+.editors img,
+.editors p {
+   /* display: inline-block; */
+   float: left;
+   cursor: pointer;
+}
+
+.editors {
+    position: relative;
+    top:1300px;
+	width:100%;
+	
+}
+
+.editors p {
+    width: 125px;
+    height: 68px;
+    margin-top:0;
+    padding-bottom: 15px;
+	margin-left: 10px;
+	font-size: 14px;
+}
+
+
+
+.science {
+    position: relative;
+	top:3000px;
+	width:60%;
+	right:35px;
+}
+
+.science button{
+	padding: 10px 15px;
+    border-radius: 6px;
+    width: 100%;
+    color:white;
+    background: #6288a5;
+    font-weight: 700;
+	cursor: pointer;
+	
+}
+
+.science :nth-child(1){
+	font-weight: bold;
+	font-size: 18px;
+	display:block;
+	margin-bottom: 8px;
+}
+
+.science :nth-child(3){
+	display:block;
+	font-size: 12px;
+	color:gray;
+}
+
+.sidebars img {
+    height: 75px;
+    width: 75px;
+}
+
+#desc1 {
+    margin-top: 7px;
+}
+
+#desc2 {
+    margin-top: 17px;   
+}
+
+.editors hr {
+    width:80%;
+}
+
+#footer {
+    position: fixed;
+    bottom: 0;
+    height: 30px;
+    background: peru;
+    width: 100%;
+}
+
+
+
+/* page layout ===============*/
+.heady{
+    position:relative;
+}
+
+.heady p{
+    font-weight: 100;
+    color:gray;
+}
+
+.theory-part{
+   position:relative;
+    overflow: hidden;
+}
+
+.theory-part img {
+    width: 970px;
+    height: 98%;
+}
+
+.box{
+	position:absolute;
+	top:28%;
+    
+}
+
+.box1 {
+	left:0;
+	width:250px;
+}
+
+.box2 {
+    left:265px;
+	width:200px;
+}
+
+.box3 {
+	left:480px;
+	width:250px;
+}
+
+.box4{
+    left:750px;
+    width: 190px;
+}
+
+.boxB {
+    top: 70%;
+}
+
+.box5{
+    left:0;
+	width:250px;
+}
+.box6{
+    left:265px;
+	width:200px;
+}
+.box7{
+    left:480px;
+	width:250px;
+}
+.box8{
+    left:740px;
+    width: 190px;
+}
+
+.box span {
+    font-weight: bold;
+}
+
+.box p {
+    font-weight: 100;
+}
+
+.larry{
+	margin-bottom: 50px;
+}
+
+.youtube{
+	width:100%;
+}
+
+.stanford1{
+	color:gray;
+}
+
+.stanford2{
+	color:lightgrey;
+}
+
+.telescope{
+	width:100%;
+}
+
+.telescope1{
+	color:gray;
+}
+
+.telescope2{
+	color:lightgray;
+}
+
+.social-icons{
+    margin:0;
+    display: inline-block;
```

Comment:
if you use the float,display becomes the block,inline-block will not work in this case.

Reply:
removed the display property